src/data/science.json

```diff
@@ -12,6 +12,7 @@
       "url": "/en/science",
       "external": false
     },
+    "mobileMessage": "This page had been optimized for mobile devices, i.e. images as shown as statics figures.",
```

Comment:
```suggestion
    "mobileMessage": "This page has been optimized for mobile devices, i.e. graphics are shown as statics figures.",
```

Reply:
What do you think about: ... i.e. graphics are shown as static images. ?

Reply:
Yep, that's better @brifemu!

Reply:
Thanks, edited and pushed!